src/utils.js

```diff
@@ -89,6 +89,14 @@ function parseFoldersToGlobs(patterns, extensions = []) {
   });
 }
 
+/**
+ * @param {any[]} list
+ * @returns {boolean}
+ */
+function hasElementsOn(list = []) {
+  return list.length > 0;
+}
```

Comment:
Doesn't need a function to check the length

Reply:
Because it's just a length check, it doesn't make sense.

Reply:
I think that's all, the rest looks good.
Thanks @Gabriel-Dias-Oliveira

Reply:
Agree :)
It should be fixed now.